<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,2,2,0,6">
    <item>
     <widget class="QCheckBox" name="chkbox_top_window">
      <property name="text">
       <string>置顶窗口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbox_input">
      <property name="title">
       <string>粘贴框</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="ptext_input"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkbox_auto_copy">
           <property name="text">
            <string>自动复制</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkbox_auto_clear">
           <property name="text">
            <string>自动清除</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbox_output">
      <property name="title">
       <string>解析结果</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="ptext_output"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save_output">
         <property name="text">
          <string>保存到表格中</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbox_save">
      <property name="title">
       <string>已保存数据</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tbl_save">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_clear_tbl">
           <property name="text">
            <string>清空表格</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_save2txt">
           <property name="text">
            <string>保存成txt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_save2xls">
           <property name="text">
            <string>保存成excel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
